service/src/main/java/bio/terra/drshub/services/DrsResolutionService.java

```diff
@@ -150,6 +149,7 @@ private DrsMetadata fetchObject(
     }
 
     if (drsResponse != null) {
+      drsMetadataBuilder.drsResponse(drsResponse);
```

Comment:
This seems fine although since the parameter `drsResponse` is declared `@Nullable` in the generated model it seems like it could work either way. Did changing this fix an error?

Reply:
It resolved a Spotbugs complaint, a bit more info in the commit message: https://github.com/DataBiosphere/terra-drs-hub/pull/97/commits/b2652ba5df4ce16796851caa31f7ec09dbda8934

Reply:
Thanks for the explanation. My impression is that spotbugs doesn't provide much if any benefit beyond what sonar is also doing. But until spotbugs is removed it makes sense to fix its warnings.

Reply:
Thanks for the explanation. My impression is that spotbugs doesn't provide much if any benefit beyond what sonar is also doing. But until spotbugs is removed it makes sense to fix its warnings.